launcher-implementation/src/main/scala/xsbt/boot/BootConfiguration.scala

```diff
@@ -46,6 +46,17 @@ private[boot] object BootConfiguration {
   /** The class name prefix used to hide the Ivy classes used by this loader from the application*/
   final val IvyPackage = "org.apache.ivy."
 
+  /** The class name prefixes used to hide Coursier and its dependencies from the application */
+  final val CoursierPackages = Array(
+    "coursier",
+    "shapeless",
+    "argonaut",
+    "org.fusesource",
```

Comment:
Not sure if I should filter this one out or not?

Reply:
See https://github.com/sbt/launcher/pull/81. You might have to check using sbt 0.13.18 on Windows.
In general it might be good to migrate to GitHub Actions for CI so we can test on Windows.